pom.xml

```diff
@@ -75,6 +75,7 @@
         <module>foxtrot-core</module>
         <module>foxtrot-server</module>
         <module>foxtrot-sql</module>
+        <module>foxtrot-translator</module>
```

Comment:
The order is wrong. Fix the ordering of modules in the order of inclusion. As a matter of fact, the sql one should come before server as well.

Reply:
Done